package.json

```diff
@@ -6,16 +6,20 @@
   "description": "A minimal GitHub App that integrates with Slack",
   "main": "index.js",
   "scripts": {
-    "dev": "node --env-file .env scripts/dev-server.js | pino-pretty --singleLine",
+    "dev:github": "node --env-file .env scripts/dev-server.js | pino-pretty --singleLine",
+    "dev:slack": "node  --env-file .env scripts/start-slack.js | pino-pretty --singleLine",
```

Comment:
This shouldn't be a separate server, we want to run everything in one process locally. In production we can have different functions that are invoked separately for github and slack